misc/stackblitz-gen.js

```diff
@@ -57,15 +58,16 @@ function generateIndexHtml() {
   return `<!DOCTYPE html>
 <html>
 
-  <head>
+  <head>xx
     <title>ng-bootstrap demo</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/${versions.bootstrap}/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.15.0/themes/prism.css" />
```

Comment:
Why?

Reply:
Because of a snippet of code I have in the demo

<img width="802" alt="image" src="https://user-images.githubusercontent.com/1152740/46534102-5a026480-c8a7-11e8-80f1-ba0fbab09ce7.png">